packages/editor/src/editor-ui/plugin-toolbar/text-controls/utils/blank.ts

```diff
@@ -50,6 +51,10 @@ function removeBlanks(editor: SlateEditor) {
       },
     })
   })
+  // After the blank is removed, we must refocus the editor. Without the timeout
+  // it blows up and moving this code to the unmounting of the blank-renderer
+  // also blows up when changing from preview to edit mode.
+  setTimeout(() => ReactEditor.focus(editor), 1)
```

Comment:
this would probably also work, right?
```suggestion
  setTimeout(() => ReactEditor.focus(editor))
```
I prefer to use `0` or no value to make it clear that it does not have to be exactly `1ms` but just not in the current thread.

Reply:
I do think so and will add some e2e tests to confirm!